src/feasibility_form_nls.jl

```diff
@@ -85,7 +85,34 @@ function FeasibilityFormNLS(nls :: FeasibilityResidual)
                      )
   nls_meta = NLSMeta(nequ, nvar, x0=[meta.x0; zeros(nequ)], nnzj=nequ, nnzh=0)
 
-  nlp = FeasibilityFormNLS(meta, nls_meta, nls, NLSCounters())
+  nlp = FeasibilityFormNLS{FeasibilityResidual}(meta, nls_meta, nls, NLSCounters())
+  finalizer(nlp -> finalize(nlp.internal), nlp)
+
+  return nlp
+end
+
+function FeasibilityFormNLS(nls :: LLSModel; name="$(nls.meta.name)-ffnls")
+  nequ = nls.nls_meta.nequ
+  meta = nls.meta
+  nvar = meta.nvar + nequ
+  ncon = meta.ncon + nequ
+  nnzh = nls.nls_meta.nnzh + nequ + (meta.ncon == 0 ? 0 : meta.nnzh) # Some indexes can be repeated
+  meta = NLPModelMeta(nvar, x0=[meta.x0; zeros(nequ)],
+                      lvar=[meta.lvar; fill(-Inf, nequ)],
+                      uvar=[meta.uvar; fill( Inf, nequ)],
+                      ncon=ncon,
+                      lcon=[zeros(nequ); meta.lcon],
+                      ucon=[zeros(nequ); meta.ucon],
+                      y0=[zeros(nequ); meta.y0],
+                      lin=[nls.nls_meta.lin; meta.lin .+ nequ],
+                      nln=[nls.nls_meta.nln; meta.nln .+ nequ],
+                      nnzj=meta.nnzj + nls.nls_meta.nnzj + nequ,
+                      nnzh=nnzh,
+                      name=name
+                     )
+  nls_meta = NLSMeta(nequ, nvar, x0=[meta.x0; zeros(nequ)], nnzj=nequ, nnzh=0, lin=1:nequ, nln=Int[])
+
+  nlp = FeasibilityFormNLS{typeof(nls)}(meta, nls_meta, nls, NLSCounters())
```

Comment:
As far as I can tell, this is identical to the generic constructor?!

Reply:
You're right, since LLSModel declares `nnzh = 0`, this can be removed.